content/src/content/jcr_root/apps/core/wcm/sandbox/components/carousel/v1/carousel/README.md

```diff
@@ -63,6 +63,28 @@ data-cmp-hook-carousel="next"
 data-cmp-hook-carousel="indicator"
 ```
 
+### Enabling Carousel Editing Functionality
+The following properties and child nodes are required to enable full editing functionality for the Carousel:
```

Comment:
@jckautzmann, @vladbailescu - added docs section on the requirements in the proxy component for a Carousel